tgstation.dme

```diff
@@ -1,6 +1,8 @@
 // DM Environment file for tgstation.dme.
 // All manual changes should be made outside the BEGIN_ and END_ blocks.
 // New source code should be placed in .dm files: choose File/New --> Code File.
+#include "code\genesis_call.dme"
```

Comment:
please add a comment explainign why this is here and that it shouldnt' be moved

Reply:
Adding a comment on the same line confuses DreamMaker, even tho it doesn't actually remove it. 
![image](https://github.com/user-attachments/assets/e7dae224-d0d4-4461-a46d-bc24db7348e9)